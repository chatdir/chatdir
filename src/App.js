import React, { Component } from 'react'
import './App.css'
import PhoneText from './img/phoneText.jpg'

class App extends Component {
  state = {}
  render() {
    return (
      <div className="AppClass">
        Hello, world!
        <div>
          <img src={PhoneText} alt="example chat on a phone" />
        </div>
      </div>
    )
  }
}

export default App
