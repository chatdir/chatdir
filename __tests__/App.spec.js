import React from 'react'
import { create } from 'react-test-renderer'
import App from '../src/App'

describe('App', () => {
  test('snapshot', () => {
    const input = create(<App />)
    expect(input.toJSON()).toMatchSnapshot()
  })
})
