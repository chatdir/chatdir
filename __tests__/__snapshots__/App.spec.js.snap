// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot 1`] = `
<div
  className="AppClass"
>
  Hello, world!
  <div>
    <img
      alt="example chat on a phone"
      src="test-file-stub"
    />
  </div>
</div>
`;
